Installing and configuration ORM

AveGus opened the pull request:
*Веб-сервер сменён с uvicorn на granian
*Проинициализирован Piccolo в качестве ORM(piccolo_conf, piccolo_app) 
*Добавлена простая модель User(для теста)
*Для удобства был добавлен pydantic_settings(settings.py) 
*Изменена архитектура

AveGus committed:
*Веб-сервер сменён с uvicorn на granian
*Проинициализирован Piccolo в качестве ORM(piccolo_conf, piccolo_app)
*Добавлена простая модель User(для теста)
*Для удобства был добавлен pydantic_settings(settings.py)
*Изменена архитектура

SiMiZZZ commented on src/config/settings.py:
```diff
@@ -0,0 +1,14 @@
+class Settings(BaseSettings):
+    model_config = SettingsConfigDict(env_file=".env", env_file_encoding="utf-8")
+    DB_HOST: str = "127.0.0.1"
+    DB_PORT: int = 5432
+    DB_USER: str = "postgres"
+    DB_PASS: str = "postgres"
+    DB_NAME: str = "postgres"
```
все должно быть вынесено в конфиг
когда до "прода" дайдем, тоже сюда креды писать открыто будем? :)

SiMiZZZ commented on src/models/users.py:
```diff
@@ -0,0 +1,10 @@
+    id = columns.UUID(primary_key=True, default=uuid.uuid4)
+    email = columns.Varchar(unique=True)
+    name = columns.Varchar()
+    password = columns.Secret(length=255)  # type: ignore
```
что за type: ignore

AveGus replied:
мупу очень сильно пытался мне доказать что секрет вот здесь это вообще злодеяние чистой воды, говорил то что я вообще бездарь и гнить мне в аду, поэтому пришлось добавить игнор

SiMiZZZ replied:
Предлагаю разобраться что именно не нравится майпаю, возможно проблема реальная. так сразу в игнор кидать не лучшая затея

AveGus committed:
В README изменен веб-сервер для запуска приложения
В users.py была явна указана игнорируемая ошибка(mypy)
Установлена зависимость для granian(чтобы была возможность использовать флаг --reload)
